src/components/formCovid/index.tsx

```diff
@@ -150,6 +148,7 @@ const FormCovid = (): FC => {
             id={id}
             dosis={dosis}
             group={group}
+            endDate={undefined}
```

Comment:
Als je een prop niet meegeeft is die toch automatisch `undefined`? Lijkt me overbodig om hier expliciet `undefined` mee te geven. Dan moet je typing in de component `endDate?: string;` zijn. Dan is die property niet verplicht mee te geven